alacritty/src/main.rs

```diff
@@ -205,7 +205,19 @@ fn alacritty(options: Options) -> Result<(), Box<dyn Error>> {
     // FIXME patch notify library to have a shutdown method.
     // config_reloader.join().ok();
 
-    // Without explicitly detaching the console cmd won't redraw it's prompt.
+    // Error reporting on Windows.
+    // If an error has occurred, show it in on the console and in a message box.
+    // It is done before FreeConsole() so the error appears on the console.
+    #[cfg(windows)]
+    if let Err(ref err) = result {
+        eprintln!("Error: {}", err);
+        if let Some(source) = err.source() {
+            eprintln!("Source: {}", source);
+        }
+        panic::error_message_box(&err.to_string(), "An error has occurred");
+    }
```

Comment:
The point was to consistently show this message box for all errors encountered, so it should be not here, but in `fn main` where the all errors are being returned, though, I'd assume only for `alacritty` itself we need a message box.

Reply:
I've removed that code.